tests/test_max_stable_set.py

```diff
@@ -0,0 +1,582 @@
+                          'id': 'link_8_7_car'},
+         'link_8_9_car': {'length': 1, 'modes': ['car'], 'freespeed': 1, 'from': 'node_8', 'to': 'node_9',
+                          'id': 'link_8_9_car'},
+         'link_9_8_car': {'length': 1, 'modes': ['car'], 'freespeed': 1, 'from': 'node_9', 'to': 'node_8',
+                          'id': 'link_9_8_car'}
+         }, silent=True)
+
+    n.schedule = Schedule(epsg='epsg:27700',
+                          services=[
+                              Service(id='bus_service',
+                                      routes=[
+                                          Route(id='service_1_route_1',
+                                                route_short_name='',
+                                                mode='bus',
+                                                stops=[Stop(epsg='epsg:27700', id='stop_1', x=1, y=2.5),
+                                                       Stop(epsg='epsg:27700', id='stop_2', x=2, y=2.5),
+                                                       Stop(epsg='epsg:27700', id='stop_3', x=5.5, y=2)],
+                                                trips={'trip_id': ['trip_1'],
+                                                       'trip_departure_time': ['15:30:00'],
+                                                       'vehicle_id': ['veh_bus_0']},
+                                                arrival_offsets=['00:00:00', '00:02:00', '00:05:00'],
+                                                departure_offsets=['00:00:00', '00:03:00', '00:07:00']
+                                                ),
+                                          Route(id='service_1_route_2',
+                                                route_short_name='',
+                                                mode='bus',
+                                                stops=[Stop(epsg='epsg:27700', id='stop_3', x=5.5, y=2),
+                                                       Stop(epsg='epsg:27700', id='stop_2', x=2, y=2.5),
+                                                       Stop(epsg='epsg:27700', id='stop_1', x=1, y=2.5)],
+                                                trips={'trip_id': ['trip_2'],
+                                                       'trip_departure_time': ['16:30:00'],
+                                                       'vehicle_id': ['veh_bus_1']},
+                                                arrival_offsets=['00:00:00', '00:02:00', '00:05:00'],
+                                                departure_offsets=['00:00:00', '00:03:00', '00:07:00']
+                                                )
+                                      ])])
+    return n
+
+
+@pytest.fixture()
+def network_spatial_tree(network):
+    return spatial.SpatialTree(network)
+
+
+def test_all_stops_have_nearest_links_returns_False_with_missing_closest_links(mocker, network, network_spatial_tree):
```

Comment:
How about `test_detects_stops_that_lack_nearest_links` or something that reads more like a description of behaviour and is not coupled to a method name?